market/demos/graphql/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
+	"id": "graphql-demo",
+	"name": "GraphQL Demo",
+	"description": "Sample GraphQL endpoint that allows clients to query for tasks and users.",
+	"type": "solution",
+	"platformReview": "4.5",
+	"sourceUrl": "https://github.com/axonivy-market/graphql-demo",
+	"statusBadgeUrl": "https://github.com/axonivy-market/graphql-demo/actions/workflows/ci.yml/badge.svg",
+	"language": "English",
+	"industry": "Cross-Industry",
+	"tags": ["demo"],
+	"validate": true,
```

Comment:
I think this validate flag is pretty internal. Means it will be downloaded by the RCPTT build and verified to have no errors in it. 
Ok for me to add it there ... but just hat you know :)